consensus/module.go

```diff
@@ -194,6 +195,8 @@ func (m *consensusModule) Start() error {
 			consensusTelemetry.CONSENSUS_BLOCKCHAIN_HEIGHT_COUNTER_DESCRIPTION,
 		)
 
+	m.logger = logger.Global.CreateLoggerForModule(m.GetModuleName())
```

Comment:
In the README, you specified that we should create this in `Start`, but here we are creating it in `Create`.

I realize it doesn't matter but just want to be consistent and set a good pattern to follow for other devs.

Can you pick up and update everywhere?

Reply:
@Olshansk from what I see you've left a comment in `Start()`.

<img width="807" alt="image" src="https://user-images.githubusercontent.com/4950477/216730297-605f97b8-6c5a-4e04-8f99-e9153f2bf057.png">

And I've checked other places `CreateLoggerForModule` is called - they are also under `Start()`.